data/pathfinder/dreamscarred_press/ultimate_psionics/up_classes.lst

```diff
@@ -695,7 +695,7 @@ CLASS:Body Snatcher	STARTSKILLPTS:2	CSKILL:Autohypnosis|Bluff|Diplomacy|TYPE=Per
 # Class Name		Hit Dice	Type					Abbreviation	Max Level	Source Page		Combat bonus										Save bonus																						Modify VAR
 CLASS:Cerebremancer	HD:6	BONUS:VAR|ClassBABPoor|classlevel("APPLIEDAS=NONEPIC")|PREVAREQ:UseFractionalBAB,1		TYPE:PC.Prestige.Psionic	FACT:Abb|Cer		MAXLEVEL:10	SOURCEPAGE:p.353	BONUS:COMBAT|BASEAB|classlevel("APPLIEDAS=NONEPIC")/2|TYPE=Base.REPLACE|PREVAREQ:UseAlternateBABProgression,0	BONUS:SAVE|BASE.Will|(classlevel("APPLIEDAS=NONEPIC")+1)/2|PREVAREQ:UseAlternateSaveProgression,0	BONUS:SAVE|BASE.Reflex,BASE.Fortitude|(classlevel("APPLIEDAS=NONEPIC")+1)/3|PREVAREQ:UseAlternateSaveProgression,0	BONUS:VAR|PsiCrystalLVL|CL|PREABILITY:1,CATEGORY=FEAT,Psicrystal Affinity	BONUS:VAR|ClassSavePoor_Fortitude|classlevel("APPLIEDAS=NONEPIC")|PREVAREQ:UseFractionalSave,1	BONUS:VAR|ClassSavePoor_Reflex|classlevel("APPLIEDAS=NONEPIC")|PREVAREQ:UseFractionalSave,1	BONUS:VAR|ClassSaveGood_Will|classlevel("APPLIEDAS=NONEPIC")|PREVAREQ:UseFractionalSave,1
 # Class Name		Required Skill								Req. Spell Type
-CLASS:Cerebremancer	PRESKILL:2,Knowledge (Arcana)=3,Knowledge (Psionics)=3	PRETEXT:PRESPELLTYPE:1,Arcane=2; PRESPELLTYPE:1,Psionic=2
+CLASS:Cerebremancer	PRESKILL:2,Knowledge (Arcana)=3,Knowledge (Psionics)=3	PRESPELLTYPE:1,Arcane=2		PRESPELLTYPE:1,Psionic=2
```

Comment:
Can you verify the syntax is correct. I believe this should be PRESPELLCAST: not TYPE, based upon the use of level flag. Which is likely why it was placed inside the PRETEXT during the conversion process.